include/boost/url/scheme.hpp

```diff
@@ -150,6 +150,16 @@ BOOST_URL_DECL
 string_view
 to_string(scheme s) noexcept;
 
+/** Return the default port for a known scheme
+
+    @return An integer with the default port number
+
```

Comment:
I mean... an easy documentation victory is to make a list `@li` with each scheme and the port number it returns.